<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #c2edce;</string>
  </property>
  <widget class="QLineEdit" name="searchbar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 246, 242);
border-radius: 10px;
padding: 8px</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Search items here</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="products_scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>491</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f6f6f2;
border-radius:10px;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>478</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:'#f6f6f2';
border-color:1px solid white;
border-width: 1px 1px 1px 0px;
color: #6fb3b8;
border-radius: 10px;
}
QPushButton:hover
{
background-color:#F8C471;
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cart_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(111, 179, 184);
border-radius:10px;
color:#372b25;
	color: rgb(255, 255, 127);
}
QPushButton:hover
{
background-color:#388087;
}</string>
   </property>
   <property name="text">
    <string>Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(111, 179, 184);
border-radius:10px;
color:#372b25;
	color: rgb(255, 255, 127);
}
QPushButton:hover
{
background-color:#388087;
}</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(184, 0, 138);
padding: 4px;
</string>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_price_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 4px;
border: rgb(194, 237, 206);
border-radius: 10px;
background-color:rgb(246, 246, 242)</string>
   </property>
   <property name="placeholderText">
    <string>Min Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_price_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 4px;
border: rgb(194, 237, 206);
border-radius: 10px;
background-color:rgb(246, 246, 242)</string>
   </property>
   <property name="placeholderText">
    <string>Max Price</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
